std/event/fs.zig

```diff
@@ -719,6 +719,16 @@ pub const WatchEventId = enum {
     Delete,
 };
 
+fn eqlString(a: []const u16, b: []const u16) bool {
+    if (a.len != b.len) return false;
+    if (a.ptr == b.ptr) return true;
+    return mem.compare(u16, a, b) == .Equal;
```

Comment:
I think it would be good to make this a constant-time compare.

Reply:
Are there security considerations on the event loop ? And why this one particularly ?